infra-templates/windows-network-services/0/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '2'
+services:
+  windows-metadata:
+    cap_add:
+    - NET_ADMIN
+    image: llparse/metadata:dev_windows
```

Comment:
Similar to what we've done in newer templates, let's move the images higher in the file and create templating for them so when we bump up the image it's easy to find/track. Here's an example: https://github.com/rancher/rancher-catalog/blob/v1.6-development/infra-templates/k8s/39/docker-compose.yml.tpl#L2